structure/docs/5-pkg-by-feature-pros-cons.md

```diff
@@ -0,0 +1,12 @@
+## Package by feature/component
+
+- Pros
+    - Fixes package by type/layer Cons
+- Cons
```

Comment:
here `Cons` is second, but in previous `Cons` was first item

Reply:
in lecture is will be from top to bottom, work with layered app, see Cons, mention some Pros, then refactor to feature and see Pros, then mention Cons